source/getting-started/installation-raspberry-pi-all-in-one.markdown

```diff
@@ -42,11 +41,12 @@ The All-In-One installer script will do the following automatically:
 *  Add both Home Assistant and Mosquitto to systemd services to start at boot
 
 
+To upgrade the All-In-One:
+*  Login to Raspberry Pi ```ssh pi@your_raspberry_pi_ip```
+*  Change to hass user `sudo su -s /bin/bash hass`
+*  Change to virtual enviroment `source /srv/hass/hass_venv/bin/activate`
+*  Update HA `pip3 install --upgrade homeassistant`
 
-Windows Users - Please note that after running the installer, you will need to modify a couple settings allowing you to "switch users" to edit your configuration files. The needed change within WinSCP can be seen here: [Imgur](http://i.imgur.com/tlOljo6.jpg)
 
-To upgrade the All-In-One:
-* Login to rPi ssh pi@IPADRESS
-* Change to hass user `sudo su -s /bin/bash hass`
-* Change to virtual enviroment `source /srv/hass/hass_venv/bin/activate`
-* Update HA `sudo pip3 install --upgrade homeassistant`
+
+*Windows Users* - Please note that after running the installer, you will need to modify a couple settings allowing you to "switch users" to edit your configuration files. The needed change within WinSCP can be seen here: [Imgur](http://i.imgur.com/tlOljo6.jpg)
```

Comment:
I know it's not your change but we should never allow imgur images. Our documentation need to be self contained